### ISSUE TITLE
LPIPS Surges After ~25k Steps and Validation Outputs Stop Updating After ~1300 Steps

### ISSUE THREAD

**Issue by Richal13Yu** [Intern-Nexus/gs-lrm-unofficial#5]
**Problem Description:**
	1.	Training on a single Titan GPU with batch_size=1 (dataset: re10k_subset) due to memory constraints
	2.	Despite trying various learning rates (e.g. 5e-6, 1e-5) and weight decay settings, LPIPS suddenly spikes around 28,000 training steps
	3.	In Weights & Biases, the validation metrics (val/psnr, val/ssim, val/lpips) and rendered images (comparison, projection, cameras) update normally at first but then stop refreshing after about 1300 steps

**Reproduction Steps:**
	1.	Configure Hydra (main.yaml) with:
+experiment=re10k
data_loader.train.batch_size=2   
optimizer.lr=5e-6   
optimizer.warm_up_steps=20000   
trainer.gradient_clip_val=1.0   
trainer.val_check_interval=10   
	2.     Launch training:
python3 -m src.main   +experiment=re10k   data_loader.train.batch_size=2   optimizer.lr=5e-6   optimizer.warm_up_steps=20000   trainer.gradient_clip_val=1.0   trainer.val_check_interval=10   
	3.	Observe W&B:
– Training loss and validation curves appear correctly initially
– At step ≈ 300 after lauched the checkpoints no further val/* logs or images are recorded
– At global_step ≈ 25,000, LPIPS on the training loss chart suddenly increases dramatically

**Expected Behavior:**
– LPIPS should gradually decrease or fluctuate within a reasonable range throughout training
– Validation metrics and images should be logged and updated every 10 steps

**Actual Behavior:**
– LPIPS remains stable until ~28k steps, then abruptly rises
– Validation metrics and rendered outputs cease updating after ~1.3k steps, despite continuing training (training loss still logs beyond that)

**Environment:**
– GPU: NVIDIA Titan (single GPU)
– Batch size: 1/2
– Dataset: re10k_subset
– Hydra config: val_check_interval = 10, gradient_clip_val = 1.0
– PyTorch Lightning settings: devices = “auto”, strategy = “auto”, (optional) max_steps = 50000

**Solutions I have tried but not working:**
	1.	Gradient Accumulation: use Trainer(accumulate_grad_batches=N) to simulate larger batch sizes and smooth updates
	2.	Learning Rate Scheduling: add a warm-up phase or switch to a cosine/step scheduler; reduce LR dynamically when LPIPS starts to spike
	3.	Weight Decay Tuning: assign different weight_decay values to bias/norm layers vs. other parameters; try smaller decay (e.g. from 1e-3 down to 1e-4)
	4.	Validation & Logging Configuration: verify val_check_interval is loaded correctly; consider using every_n_train_steps or epoch-based interval; ensure rank_zero_only decorators aren’t skipping logs
	5.	Debug Validation Hook Execution: inspect console output to confirm validation_step and image-logging callbacks continue past step 300; check for unintended early stopping, max_steps limits, or other callback interactions

**screenshot:**

<img width="1007" height="374" alt="Image" src="https://github.com/user-attachments/assets/2e9c28d2-6bd5-4c62-af81-045af7d3dce7" />

<img width="999" height="853" alt="Image" src="https://github.com/user-attachments/assets/27603477-9c5a-4c74-acc1-25bd52f5ca04" />

<img width="1239" height="370" alt="Image" src="https://github.com/user-attachments/assets/0cc76c5f-0f80-45e0-901f-50063b5c0b31" />

Any insights or additional suggestions would be greatly appreciated!
